### PR TITLE
[COST-3614] AWS category type ahead

### PR DESCRIPTION
**Pull request by myersCody**
## Jira Ticket

[COST-####](https://issues.redhat.com/browse/COST-####)

## Description

This change will ...

## Testing

1. Checkout Branch
2. Restart Koku
3. Hit endpoint or launch shell
    1. You should see ...
4. Do more things...

## Notes

...

### CODE REVIEW

File: koku/api/resource_types/aws_category/serializers.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#
+# Copyright 2023 Red Hat Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Serializer for aws category resource types."""
+import logging
+
+from rest_framework import serializers
+
+LOG = logging.getLogger(__name__)
```
Comment (maskarb) [inline koku/api/resource_types/aws_category/serializers.py:10]: We should not import logging and create this logger if we are not going to use it. If you use this locally for debugging, that is fine if that is how you work, but don't leave it here. It is unused code.

File: koku/api/resource_types/test/aws_category/test_views.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    @RbacPermissions({"aws.account": {"read": ["*"]}})
+    def test_aws_categories_view_with_wildcard(self):
+        """Test aws categories return."""
+        with schema_context(self.schema):
+            expected = AWSEnabledCategoryKeys.objects.filter(enabled=True).values_list("key", flat=True)
+            url = reverse("aws-categories")
+            response = self.client.get(url, **self.headers)
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+            json_result = response.json()
+            LOG.info(json_result)
```
Comment (maskarb) [inline koku/api/resource_types/test/aws_category/test_views.py:44]: We don't need to log anything in unit tests, so let's remove these and the logger.

**Review by maskarb — COMMENTED** [review]
This PR looks pretty good overall. I have a couple of nitpicks:

File: koku/api/resource_types/aws_category/serializers.py (hunk 1 above)
Comment (myersCody) [reply koku/api/resource_types/aws_category/serializers.py:10]: This was honestly just a sloppy copy & paste error, so I went back to where I copied it from and removed it from there. Which made me wonder where else we import logging and don't use it. Which lead me down a rabbithole, but I think I got all instances where we import it and don't use it.

File: koku/api/resource_types/test/aws_category/test_views.py (hunk 2 above)
Comment (myersCody) [reply koku/api/resource_types/test/aws_category/test_views.py:44]: Same as above, your comment lead me down an audit and I removed the logger from the unittest files.

**Review by maskarb — APPROVED** [review]
lgtm! Thanks for removing all those instances of the unused logger!

**Comment by myersCody** [comment]
successful smokes: https://ci.ext.devshift.net/job/project-koku-koku-pr-check/9030/

**Review by maskarb — APPROVED** [review]

**Pull request merged**